docs/references/javascript/types/redirect-options.mdx

```diff
@@ -11,6 +11,6 @@ An interface that provides options for a range of redirect methods.
 
 | Name | Type | Description |
 | --- | --- | --- |
-| `afterSignInUrl` | `string \| null \| undefined` | Full URL or path to navigate to after successful sign-in. |
-| `afterSignUpUrl` | `string \| null \| undefined` | Full URL or path to navigate to after successful sign-up. Sets the `afterSignUpUrl` if the "Sign up" link is clicked. |
-| `redirectUrl` | `string \| null \| undefined` | Full URL or path to navigate to after successful sign-in, or sign-up. The same as setting `afterSignInUrl` and `afterSignUpUrl` to the same value. |
\ No newline at end of file
+| `afterSignInUrl?` | `string \| null \| undefined` | Full URL or path to navigate to after successful sign-in. |
+| `afterSignUpUrl?` | `string \| null \| undefined` | Full URL or path to navigate to after successful sign-up. Sets the `afterSignUpUrl` if the "Sign up" link is clicked. |
+| `redirectUrl?` | `string \| null \| undefined` | Full URL or path to navigate to after successful sign-in, or sign-up. The same as setting `afterSignInUrl` and `afterSignUpUrl` to the same value. |
```

Comment:
I assume the `?` implies optional, is that stated anywhere or a legend in the docs for these things.

Reply:
I am pulling the optional operator straight from our JS repo!
If the type in the JS repo states that the prop is optional, I am reflecting that in our docs.